src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -46,9 +46,9 @@ public static GitDetail TryGetFileDetail(string filePath)
             {
                 detail = GetFileDetail(filePath);
             }
-            catch (Exception)
+            catch (Exception ex)
             {
-                // ignored
+                Logger.LogWarning(ex.ToString());
```

Comment:
>LogWarning [](start = 23, length = 10)

too much warning if git does not exists?

Reply:
OK, I was in a dilemma about this. During debugging, ex.Message wasn't enough to narrow down where the fault was.  Adding a stack trace proved critical in finding faults.

What do you think?

Reply:
If you look at the example above, the exception results in a warning:
`[17-09-26 06:27:39.978]Warning:[BuildCore.Build Document.Load.ConceptualDocumentProcessor](index.md)ApplicationName='type', CommandLine='git', CurrentDirectory='', Native error= Cannot find the specified file`

Notice that this tells you that this command fails, but doesn't let you know exactly at which point it fails.

After thinking about this, do you think it would be better to have
```
Logger.LogWarning(ex.Message);
Logger.LogDiagnostic(ex.ToString());
```